lib-opensource/emqx_management/src/emqx_mgmt_cli.erl

```diff
@@ -550,10 +550,21 @@ listeners(["stop", _Proto, ListenOn]) ->
     end,
     stop_listener(emqx_listeners:find_by_listen_on(ListenOn1), ListenOn1);
 
+listeners(["restart", Identifier]) ->
+    case emqx_listeners:restart_listener(Identifier) of
+        ok ->
+            emqx_ctl:print("Restarted ~s listener successfully.~n", [Identifier]);
+        {ok, _} ->
```

Comment:
`emqx_listeners:restart_listener/1` does not return `{ok, Pid}`
so we can delete this clause.

Reply:
ah, sorry, `restart_listner/1` may return `{ok, Pid}`